pgxn/neon/control_plane_connector.c

```diff
@@ -236,42 +238,68 @@ SendDeltasToControlPlane()
 	if (!ForwardDDL)
 		return;
 
+	if (handle == NULL)
+	{
+		static struct curl_slist *headers = NULL;
```

Comment:
```suggestion
		struct curl_slist *headers = NULL;
```
doesn't need to be 'static' AFAICS

Reply:
Yeah. I think you're right. Good call out.